mergedSchema.js

```diff
@@ -0,0 +1,46 @@
+import { mergeSchemas, introspectSchema, makeRemoteExecutableSchema } from "graphql-tools"
+import { createHttpLink } from "apollo-link-http"
+import fetch from "node-fetch"
+
+import localSchema from "./schema"
+
+export default async function mergedSchema() {
+  const convectionLink = createHttpLink({
+    fetch,
+    uri: process.env.CONVECTION_GRAPH_URL,
+    headers: {
+      Authorization: `Bearer ${process.env.CONVECTION_TOKEN}`,
+    },
+  })
+
+  const convectionSchema = await makeRemoteExecutableSchema({
+    schema: await introspectSchema(convectionLink),
+    link: convectionLink,
+  })
+
+  const linkTypeDefs = `
+    extend type Submission {
+      artist: Artist
+    }
+  `
+
+  return mergeSchemas({
+    schemas: [localSchema, convectionSchema, linkTypeDefs],
+    // Prefer others over the local MP schema.
+    onTypeConflict: (_leftType, rightType) => {
+      console.warn(`[!] Type collision ${rightType}`) // eslint-disable-line no-console
```

Comment:
@orta There’s currently a bunch of consignments related type name conflicts, presumably because you’ve manually added these to MP’s schema before:

```
19:04:06 web.1       |  [!] Type collision ID
19:04:06 web.1       |  [!] Type collision Submission
19:04:06 web.1       |  [!] Type collision String
19:04:06 web.1       |  [!] Type collision Asset
19:04:06 web.1       |  [!] Type collision JSON
19:04:06 web.1       |  [!] Type collision Boolean
19:04:06 web.1       |  [!] Type collision Int
```

The `ID`, `String`, `Boolean`, and `Int` ones are built-in though, it seems weird that `graphql-tools` would trigger this callback for those 🤔

Reply:
Yeah, that's tricky, to remove Submission entirely in metaphysics - I'd have to recreate a bunch of  mutations on convection, feasible, but time consuming

Reply:
That said, the submission type should be the exact same object - so maybe it's fine to return the convection one

Reply:
WRT the JSON type - I would assume they have the same behavior in terms of representing an `any` - I bet we get these primitives for every Ruby GraphQL

Reply:
The mutations [already exist](https://github.com/artsy/convection/blob/master/app/graph/mutations.rb) in convection-- might need some updating but the basic functionality is there.

Reply:
Ah, and indeed the mutations are already automatically available in the stitched schema, I haven’t tried them yet, though.

<img width="222" alt="screen shot 2017-11-14 at 19 16 18" src="https://user-images.githubusercontent.com/2320/32796932-6e3f9b66-c970-11e7-8ba4-f564dab015ae.png">